src/base.c

```diff
@@ -25,15 +25,18 @@ base_map(extent_hooks_t *extent_hooks, unsigned ind, size_t size) {
 	if (extent_hooks == &extent_hooks_default) {
 		addr = extent_alloc_mmap(NULL, size, PAGE, &zero, &commit);
 	} else {
+		assert(!tsdn_null(tsdn));
```

Comment:
Nit: Comment on why this assert should be safe.

Sorry for the multiple rounds on just a style thing, I'd like to raise our level of explanation for this sort of thing in general.